aws/templates/common.ftl

```diff
@@ -889,7 +889,7 @@
     [#return getOccurrencesInternal(component, tier) ]
 [/#function]
 
-[#function getLinkTarget occurrence link]
+[#function getLinkTarget occurrence link allowNotFound=false ]
```

Comment:
Not sure about this. not found is about the link target not being in the solution file - which it should be.

I think what you want is to check if the target is deployed.